This fixes a segfault that can happen on a malformed message; patch comes from
upstream http://bugs.mutt.org/3483

--- a/sendlib.c
+++ b/sendlib.c
@@ -1799,7 +1799,7 @@
   else
   {
     t = strchr (start, ':');
-    if (t > end)
+    if (!t || t > end)
     {
       dprint (1, (debugfile, "mwoh: warning: header not in "
 		  "'key: value' format!\n"));
